check/ts-lint-and-format

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
nit: add copyright notice

Reply:
Shebang line [should be always on the first line](https://stackoverflow.com/questions/12910744/why-should-the-shebang-line-always-be-the-first-line#:~:text=4%20Answers&text=The%20shebang%20must%20be%20the,file%20available%20to%20that%20program.).